plugins/applyTransformsShapes.js

```diff
@@ -0,0 +1,233 @@
+import { collectStylesheet, computeStyle } from '../lib/style.js';
+import { removeLeadingZero } from '../lib/svgo/tools.js';
+import { attrsGroupsDefaults } from './_collections.js';
+import { transform2js, transformsMultiply } from './_transforms.js';
+
+export const name = 'applyTransformsShapes';
+export const description = 'Applies transforms to some shapes.';
+
+const APPLICABLE_SHAPES = ['circle', 'ellipse', 'rect', 'image'];
+
+/**
+ * @typedef {number[]} Matrix
```

Comment:
Let's not define a type aliases for simple types like this. Instead, wherever you specify `Matrix` you can put `number[]` directly and specify in the JSDoc that it's expecting a Matrix.

Reply:
This code was based on applyTransforms.js which also does the same thing. Should I change it to `number[]` anyway?

Reply:
Ahh, that was my bad. Since I've already brought it up, yeah, I'd appreciate it if you'd still update the types to avoid a simple alias.

Later we can make the same change in `applyTransforms` as a chore, but no need to do it now.